pom.xml

```diff
@@ -2984,6 +2986,12 @@
                 <version>${lettuce.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>org.projectlombok</groupId>
+                <artifactId>lombok</artifactId>
+                <version>${lombok.version}</version>
+            </dependency>
```

Comment:
I think there's still an open question here about having scope=provided so that we exclude lombok from any assemblies since it is a build time only component.

Reply:
I added an exclusion for lombok at the pring-boot-maven-plugin level in order for lombok to stop appearing in the jars. It looks like it works for me. What do you think about it?

Reply:
I think that would work for the spring boot maven plugin -- which is fine for your current use - but as we extend the use of lombok more broadly (which is appealing) we really want an approach that works with as little explicit action by the new user as possible. I think the 'scope' does this -- and is also what seems to be recommended more generally?

I know we still have the question about the dependency checker -- but that's somewhat of a limitation in that tool - for now could add an exclusion for lombok anyway, perhaps with a comment to explain why

Reply:
Ok then, I put provided back :)

Reply:
good by me :-)